v2/goi18n/merge_command_test.go

```diff
@@ -35,6 +35,50 @@ func TestMerge(t *testing.T) {
 				"active.en-US.toml": []byte("1HelloMessage = \"Hello\"\n"),
 			},
 		},
+		{
```

Comment:
Can you reduce this to two tests total (one from this hunk of the diff and also the plural one from the last hunk)? The order of translations doesn't matter so the extra test cases don't really do anything.